webpack.config.js

```diff
@@ -9,6 +9,7 @@ var base = {
         host: '0.0.0.0',
         port: process.env.PORT || 8073
     },
+    devtool: 'source-map',
```

Comment:
I tried a few different options (certainly not the whole set) and here's what I saw:
- omit `devtool` entirely: no source maps, but `scratch-vm.js` is fairly readable.
- `source-map`: debugging and mapping to workspace files both work.
- `cheap-module-eval-source-map`: works well for debugging but appends `?abcd` (four hex digits) to most file names, which seems to prevent Chrome from mapping to workspace files.
- `eval-source-map`: effectively the same as `cheap-module-eval-source-map`.
- `eval`: gives good file names but that's about it. Functions in `scratch-vm.js` are no longer readable. I tried this because one of our other repositories uses it.

A few web searches seemed to suggest that updating webpack to 1.14 or newer would fix the `?abcd` thing. My vote is to stick with `source-map` for the moment and dig a little deeper (including a webpack upgrade) as a separate adventure.